packages/performance/src/controllers/perf.test.ts

```diff
@@ -38,6 +42,16 @@ describe('Firebase Performance Test', () => {
     options: fakeFirebaseConfig
   } as unknown) as FirebaseApp;
 
+  describe('#constructor', () => {
+    it('does not initialize performance if the required apis are not available', () => {
+      stub(Api.prototype, 'requiredApisAvailable').returns(false);
+      stub(initializationService, 'getInitializationPromise');
+      new PerformanceController(fakeFirebaseApp);
+
+      expect(initializationService.getInitializationPromise).not.be.called;
```

Comment:
Add a check on the existence of console log?

For example:
expect( console.log.calledWith('Firebase Performance cannot start if browser does not support fetch and Promise or cookie is disabled.') ).to.be.true;

Reply:
Added a check for the consoleLogger to be called. Not sure about adding the whole message though.